yahoo_panoptes/framework/resources.py

```diff
@@ -837,12 +844,12 @@ def get_resources(self, query):
                 if row[8] and row[9]:
                     metadata_keys = row[8].split('|')
                     metadata_values = row[9].split('|')
-                    for i in range(0, len(metadata_keys) - 1):
```

Comment:
OBOE

Reply:
My preferred woodwind instrument is an English Horn, but I can be convinced of looking into an Oboe...